Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -135,10 +132,21 @@ private bool ShouldMoveIssuesToNextVersion (string versionID, string nextVersion
       return false;
 
     _console.WriteLine("These are some of the issues that will be moved by releasing the version on jira:");
-    foreach (var issue in issuesToMove.Take(5))
-      _console.WriteLine($"'{issue.Key} - {issue.Fields.Summary}'");
+    PrintIssueListing(issuesToMove);
     _console.WriteLine("Do you want to move these issues to the new version and release the old one or just release the old version?");
 
     return _inputReader.ReadConfirmation();
   }
+
+  private void PrintIssueListing (IReadOnlyList<JiraToBeMovedIssue> issues, int displayAmount = 5, int maxLength = 75)
+  {
+    foreach (var issue in issues.Take(displayAmount))
+    {
+     var summary = issue.Fields.Summary.Length <= maxLength
+       ? issue.Fields.Summary
+       : issue.Fields.Summary.Substring(0, maxLength - "...".Length) + "...";
+      _console.WriteLine($"* '{issue.Key} - {summary}'");
```

Comment:
It would be easier to first concat and then truncate. that way, we apply the max length to the entire line.